startUp.js

```diff
@@ -1,11 +1,49 @@
 (function () {
     "use strict";
-    var express = require('express');
-    var twitter = require('./controller/twitter');
+    let express = require('express');
+    let mongoose = require ('mongoose');
+    let config = require('./config/config');
+    let mongodbConnection = "mongodb://";
 
+    let twitterStreaming = require('./controller/twitter');
+    let loadBasicPokemonDetails = require('./controller/loadBasicPokemonDetails');
+
+    //get database String
+    function getDbString(database) { 
+        return mongodbConnection + database.uri + ":" + database.port + "/" + database.collection;
+    }
+
+    function setupDataBase() {
+        console.log("Connecting to " + getDbString(config.database));
+        mongoose.connect(getDbString(config.database));
+        let db = mongoose.connection;
+
+        // MongoDB events
+        db.on('connected', function () { 
+            console.log('Mongoose connected');
+        });
+        db.on('error', function (err) {
+            console.log('Mongoose default connection error: ' + err);
+        });
+        db.on('disconnected', function () {
+            console.log('Mongoose default connection disconnected');
+        });
+        db.on('open', function () {
+            console.log("Mongoose connection open");
+        });
+    }
+    
```

Comment:
@samitsv hey, maybe you could review my commit on the db setup a11a5480, as it also involves basic workflow for storing of the pokemon details and remove this part from here.